src/deviceApiCalls/deviceApiCalls.json

```diff
@@ -14,5 +14,26 @@
   "getAvailableInputTypes": {
     "id": "getAvailableInputTypesResponse",
     "resultValidator": "./schemas/getAvailableInputTypes.result.json"
+  },
+  "getAutofillInitData": {
+    "id": "getAutofillInitDataResponse",
+    "description": "This is called inside an overlay (eg: on Windows or soon also on macOS) to retrieve available data",
+    "resultValidator": "./schemas/getAutofillInitData.result.json"
+  },
+  "getAutofillCredentials": {
+    "id": "getAutofillCredentialsResponse",
+    "description": "Used to retrieve a specific set of credentials",
+    "paramsValidator": "./schemas/getAutofillCredentials.params.json"
+  },
+  "setSize": {
+    "description": "Used by Windows to communicate the desired size of the overlay to the native side",
```

Comment:
Wasn't this used by macOS as well?

Reply:
Not yet, macOS hasn't migrated to these calls yet (which was done to minimize changes)